src/test/javascript/spec/component/text-assessment-dashboard/text-assessment-dashboard.component.spec.ts

```diff
@@ -0,0 +1,253 @@
+    let sortService: SortService;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            declarations: [
+                TextAssessmentDashboardComponent,
+                MockComponent(AssessmentFiltersComponent),
+                MockComponent(AssessmentWarningComponent),
+                MockComponent(ResultComponent),
+                MockDirective(HasAnyAuthorityDirective),
+                MockDirective(TranslateDirective),
+                MockDirective(SortDirective),
+                MockRouterLinkDirective,
+                MockQueryParamsDirective,
+                TranslatePipeMock,
+                MockPipe(ArtemisDatePipe),
+            ],
+            providers: [
+                JhiLanguageHelper,
+                { provide: AccountService, useClass: MockAccountService },
+                { provide: ExerciseService, useClass: MockExerciseService },
+                MockProvider(TextSubmissionService),
+                MockProvider(TextAssessmentService),
+                MockProvider(SortService),
+                {
+                    provide: ActivatedRoute,
+                    useValue: {
+                        snapshot: {
+                            paramMap: convertToParamMap({
+                                exerciseId: textExercise.id,
+                                examId: 2,
+                            }),
+                        },
+                    },
+                },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(TextAssessmentDashboardComponent);
+                component = fixture.componentInstance;
+                exerciseService = fixture.debugElement.injector.get(ExerciseService);
+                textSubmissionService = fixture.debugElement.injector.get(TextSubmissionService);
+                textAssessmentService = fixture.debugElement.injector.get(TextAssessmentService);
+                accountService = fixture.debugElement.injector.get(AccountService);
+                sortService = fixture.debugElement.injector.get(SortService);
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should set parameters and call functions on init', () => {
+        // setup
+        const exerciseServiceFindMock = jest.spyOn(exerciseService, 'find');
+        exerciseServiceFindMock.mockReturnValue(of(new HttpResponse({ body: textExercise })));
+        const getTextSubmissionStub = jest.spyOn(textSubmissionService, 'getTextSubmissionsForExerciseByCorrectionRound');
+        getTextSubmissionStub.mockReturnValue(of(new HttpResponse({ body: [textSubmission], headers: new HttpHeaders() })));
+
+        // test for init values
+        expect(component).toBeTruthy();
+        expect(component.submissions).toEqual([]);
+        expect(component.reverse).toEqual(false);
```

Comment:
```suggestion
        expect(component.reverse).toBe(false);
```
When comparing primitive values (such as number, string, etc.) we should use toBe.